app/src/main/kotlin/com/aliucord/manager/ui/viewmodel/InstallViewModel.kt

```diff
@@ -101,17 +101,14 @@ class InstallViewModel(
         }
 
         // Download the native libraries split
-        val libsApkFile = step(InstallStep.DL_LIBS_APK) {
+        val libsApkFile = step(librariesApkDl, StepCategory.APK_DL) {
             val libArch = arch.replace("-v", "_v")
             externalCacheDir.resolve("config.$libArch-${supportedVersion}.apk").let { file ->
-                if (file.exists()) {
-                    log += "cached... "
-                } else {
```

Comment:
some cached indicator would be nice
i thought the downloads 404'd instead when the time elapsed was 0.1s

ex: "cached, " italics + low alpha grey?
![image](https://user-images.githubusercontent.com/33725716/198446715-5a1943ef-c627-447b-bbb4-b6a3b6e9372e.png)